docs/recipes/installTips.md

```diff
@@ -365,3 +365,22 @@ During training and analysis steps, DeepLabCut does not use the GPU processor he
 ![](https://images.squarespace-cdn.com/content/v1/57f6d51c9f74566f55ecf271/b1d03ca0-f8ba-4a31-a399-6e86856c81b0/installBrandon3.png?format=500w)
 
 (5) If you don't see activity there during training, then your GPU is likely not installed correctly for DeepLabCut. Return to the installation instructions, and be sure you installed CUDA 11+, and ran `conda install cudnn -c conda-forge` after installing DeepLabCut.
+
+## How to install Deeplabcut for Intel, AMD (and Nvidia) GPUs
```

Comment:
Maybe it is better to put also in the title that it is for windows only? So that someone can easily see it with a glance

`## How to install Deeplabcut on Windows for Intel, AMD (and Nvidia) GPUs`